aspnetcore/fundamentals/servers/index.md

```diff
@@ -20,14 +20,13 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 ASP.NET Core ships two server implementations:
 
 * [Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform HTTP server based on [libuv](https://github.com/libuv/libuv), a cross-platform asynchronous I/O library.
```

Comment:
Is this a 1.x section? If not, I'd just drop everything after and including "based on".  Maybe you could mention it's the default server, but that's covered almost immediately after this already.